src/cocoa/toga_cocoa/widgets/image.py

```diff
@@ -6,8 +9,14 @@ def __init__(self, interface):
         self.interface = interface
         self.interface.impl = self
 
+    def __get_full_path(self, path):
```

Comment:
Since this method doesn't use the instance of the class at all, could we make it a static method? Also the convention used for most of the methods in Toga is to make the protected (single underscore) instead of private (double underscore), is there a reason to make this one private?